machines/vic/certificate.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
If we use the tool to generate certs then the user needs to have go installed on the system the install is run from, which is not something we can assume.
Certificate generation as part of the install is provided as a convenience - they need to be secure, but we can provide the certificate basics only and if users want to have fine control of the contents they can specify pre-generated certs to use.